traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -16,10 +16,7 @@
 package com.comcast.cdn.traffic_control.traffic_router.core.router;
 
 import java.io.IOException;
-import java.net.InetAddress;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.UnknownHostException;
+import java.net.*;
```

Comment:
This is a bit of a nitpick, but I think we should stay consistent with our imports in explicitly importing every class individually that is actually used.

Reply:
done